bundles/org.smarthomej.binding.tuya/src/main/java/org/smarthomej/binding/tuya/internal/config/ChannelConfiguration.java

```diff
@@ -25,6 +25,7 @@ public class ChannelConfiguration {
     public int dp2 = 0;
     public int min = Integer.MIN_VALUE;
     public int max = Integer.MAX_VALUE;
+    public Boolean inverted = Boolean.FALSE;
```

Comment:
```suggestion
    public boolean reversed = false;
```

Use primitives where possible. Since a default is provided, a primitive is sufficient here. I would also suggest to name the parameter "reversed", because an inversion (and that was my fault when I suggested the profile) us usually associated with a 100 -> -100 transformation.